src/base.ts

```diff
@@ -84,6 +84,7 @@ function recorderFromEnv(prefix: string): Recorder {
     case 'production':
       // NODE_ENV set to `production` in bin/run_bin, which is specified in 'bin' package.json section
       sink = new NewRelicSink('eu01xx73a8521047150dd9414f6aedd2FFFFNRAL');
+      console.log('IMPORTANT MESSAGE: We are tracking metrics anonymously from the commands you are executing just for statistical purposes.');
```

Comment:
You should better use `this.log` so we use the logger configured in Oclif. In fact, not sure if better log it as a warning, so then `this.warn` could be used.

Reply:
Additionally, I would improve the message by adding instructions on how users can disable metrics.

Reply:
I'm trying to use both `this.log` and `this.warn` but the message is not shown...weird 🤔

Reply:
> Additionally, I would improve the message by adding instructions on how users can disable metrics.

I'm thinking about adding something like:
```
If you want to disable this tracking, please set the 'ASYNCAPI_METRICS' env variable to 'false' when executing the command.
For example, run the following command instead:
>> ASYNCAPI_METRICS=false asyncapi validate spec_file.yaml
```
WDYT?

Reply:
> I'm thinking about adding something like:

Based on your message, this is what ChatGPT gave me as simplification: 

```
To disable tracking, set the 'ASYNCAPI_METRICS' env variable to 'false' when executing the command. For instance:

ASYNCAPI_METRICS=false asyncapi validate spec_file.yaml
```

Looks simpler and shorter.

Reply:
> I'm trying to use both `this.log` and `this.warn` but the warning message is not shown then...weird 🤔

They should work. The issue is that you are calling those from a function that has no `this` scope because is not part of the base command class. We either need to pass a command to that function or rather add this function as part of the base class (second is easier and could also make sense).